### DISCUSSION TITLE
[Bug] Android: Stuck in loading process when Notifications page is empty

### DISCUSSION THREAD

**Discussion by Dragon1573** [community/community#77749] [Mobile] [answered]
### Select Topic Area

Bug

### Description

Currently, no informations in my Notification Inbox.

But on GitHub Mobile for Android, the app will stuck in a loading page when "Notifications" page with specific filters are empty.

<details><summary>GitHub Notifications on Web</summary>

![image](https://github.com/community/community/assets/49941141/618491ad-72d0-41ec-939c-312368441340)

</details>
<details><summary>GitHub Notifications on Android Mobile</summary>

![Screenshot_20231203_141848_com github android](https://github.com/community/community/assets/49941141/8ced0c03-349b-41a4-90ca-40ab703a36b1)

</details>

### Environments

- HarmonyOS v4.0.0.166 (based on Android 12 aarch64)
- Dual framework (GMS + HMS Core) exists on my device
  - Google Service Framework (服务框架) v10
  - Google Account Management Program (账户管理程序) v4.4.4-1227136
  - Google Play Service v20.24.14 (120408-319035315)
  - Google Play Store v38.4.19-29 [0] [PR] 582389963
  - HMS Service Framework (服务框架) v11.1.8.301
  - HMS Core v6.12.2.302
- GitHub Mobile v1.141.0-beta (10167)
  - Installed via Google Play Store

**Comment by iskae** [comment]
Hi @Dragon1573 👋 

Thanks so much for reaching out with a detailed bug report ❤️ 

We have fixed the issue and will be rolling out the fix later this week 🙇 

Cheers!

**Reply by Dragon1573** [reply]
Thank you @iskae ❤️

Could you please give a simple explaination of how you (and your team) locate and fix this bug? I'm curious about it 😄

**Reply by iskae** [reply, marked as answer]
Sure thing! We were missing an initial state in case there were no notifications to begin with 😅 We simply added it 🙇
